### PR TITLE
Medical Changes: Increased Cloning Speed

### PR DESCRIPTION
**Pull request by jlsnow301**
## About The Pull Request
Medical Changes: Increased base cloning speed, healing percentage.

To-do: Perhaps simplify the cloner UI, reduce chance that it will clone an SSD player.

## Why It's Good For The Game
Cloning is a madhouse in BeeStation. When it's not completely jam-packed with corpses and no one to clone them, there's a sole person resurrecting half the station. It takes so long that 2/3rds of the clones are disconnected when finished. This increases the base-speed of the cloner to allow non-ssd players a better chance at getting revived this century.

## Changelog
:cl:
tweak: Base cloning speed, healing percentage.
/:cl:

### CODE REVIEW

**Comment by qwertyquerty** [comment]
Could also just have a mechanic make you more cloners

**Comment by jlsnow301** [comment]
Medbay would be lucky to even see them once a round and simultaneously NOT be a traitor

File: code/game/machinery/cloning.dm [hunk 1]
```diff
@@ -65,8 +65,8 @@
 	. = ..()
 
 /obj/machinery/clonepod/RefreshParts()
-	speed_coeff = 0
-	efficiency = 0
+	speed_coeff = 2
+	efficiency = 2
```
Comment (qwertyquerty) [inline code/game/machinery/cloning.dm:69]: what are these for

**Comment by jlsnow301** [comment]
From what I can tell: This is the base multiplier for both the speed and final health% on release of the cloning pod. It essentially starts at the first upgrade rather than 0, but just below the upgrade where it can autoprocess by default. Upgrades to the scanning modules and manipulators are additive += afterwards, meaning that you can get it going fast and release clones at nearly full health if fully upgraded (no one does this though)

**Comment by Toxici11i** [comment]
This ^, through varediting and testing I can confirm these are the efficiency(speed and damage) multipliers for cloning

**Comment by Toxici11i** [comment]
how did you even do this

**Comment by jlsnow301** [comment]
Reopened elsewhere. GITHUB DESKTOP SO GOOD (it isn't)

**Pull request closed without merging**